src/styles/dictionary_concepts.scss

```diff
@@ -6,6 +6,14 @@
   width: auto;
 }
 
+.custom-dictionary-mappings {
+  text-align: left;
+  margin-top: 0%;
```

Comment:
@dessHub, does `margin-top: 0%` have an effect on the interface styling?

Reply:
This was for testing purposes, I forgot to remove it. On it.